dts/bindings/dma/dmamux-controller.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019, STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for DMAMUX controllers
+
+include: base.yaml
+
+bus: dmamux
+
+properties:
+    label:
+      required: true
+
+    "#dma-cells":
+      type: int
+      required: false
```

Comment:
why is this false?

Reply:
none, 
setting as true like in dma-controller.yaml